recipes/gpcal/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "gpcal" %}
+{% set version = "0.1.1.26" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ca87d0418e51cb9b7334b2d48a6102c2730eeb687e4a6a5f85585e7ea98cda07
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - astropy
+    - matplotlib
+    - numpy
+    - pandas
+    - pip
+    - python
+    - scipy
```

```suggestion
  host:
    - pip
    - python
```
I believe here you just need ``python`` and ``pip``